sensors/sensor_mock.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 )
 
-// TimedLidarSensorMock represents a fake TimedLidarSensor.
+// TimedLidarSensorMock represents a fake TimedLidarSensor. (temp)
 type TimedLidarSensorMock struct {
```

Comment:
[q] Do we need to add Properties to these mock sensors? Both imu and lidar?

Note: will remove added comment before merging

Reply:
I don't think so - no need to implement mock functionality unless there's a test that actually requires it